.editorconfig

```diff
@@ -0,0 +1,13 @@
+root = true
+
+[*]
+end_of_line = lf
+insert_final_newline = true
```

Comment:
We don't want to force final nl in certain layout files, though it seems that VS Code ignores this ... ?

Reply:
I'm happy to take it out -- we'll want to back-port the change. I'm not super happy with the way editor config plugins seem to only work some of the time across platforms/editors...